README.md

```diff
@@ -26,6 +26,9 @@ Dagger Query can be used to investigate the dagger dependency graph.
     > ./dagger-query.sh YOUR_PATH allpaths com.google.Office com.google.Beach \
     > ./dagger-query.sh YOUR_PATH somepath com.google.Paris com.google.sights.EiffelTower
     
+   If permission is denied, run this command:
+    > chmod u+r+x dagger-query-ui.sh
```

Comment:
nit: `u+rx` is more idiomatic

Reply:
Actually you can make the file executable when you check it in to git (see https://stackoverflow.com/a/38285462), so that would save the user this step.

Reply:
That's incredible, thank you!

I have added executable permissions to the scripts, but let's save this lines in README if on some platforms/machines it will still require do it manually.

Reply:
Sounds good!